src/logistration/InstitutionLogistration.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+import messages from './messages';
+
+const LogistrationDefaultProps = {
+  secondaryProviders: [],
+  buttonTitle: '',
+};
+const LogistrationProps = {
```

Comment:
We should move this to the bottom where we are actually defining the `propTypes` for the components.